app/modules/assignment/actions/__tests__/assignment-fetch-info-test.js

```diff
@@ -0,0 +1,77 @@
+import { expect } from "chai"
+import * as sinon from "sinon"
+
+import { assignmentFetchInfo } from "../assignment-fetch-info"
+import {ASSIGNMENT_ERROR_INFO, ASSIGNMENT_REQUEST_INFO, ASSIGNMENT_RECEIVE_INFO} from "../../constants"
+
+const jsonOK = (body) => {
+  const mockResponse = new window.Response(JSON.stringify(body), {
+    status: 200,
+    headers: {
+      "Content-type": "application/json"
+    }
+  })
+  return Promise.resolve(mockResponse)
+}
+
+describe("assignmentFetchInfo", () => {
+  let invalidURLAssignment = {
+    name: "Test Assignment",
+    type: "individual",
+    url: "invalidURL",
+    isFetching: false,
+    error: null,
+  }
+
+  let validAssignment = {
+    name: "Test Assignment",
+    type: "individual",
+    url: "http://classroom.github.com/classrooms/test-org/assignments/test-assignment",
+    isFetching: false,
+    error: null,
+  }
+
+  let validSettings = {
+    username: "testUser",
+  }
+
+  it("dispatches error action on invalid URL", async () => {
+    const getState = () => ({ assignment: invalidURLAssignment, settings: validSettings })
+    const dispatch = sinon.spy()
+    await assignmentFetchInfo()(dispatch, getState)
+
+    // eslint-disable-next-line no-unused-expressions
```

Comment:
:art: you could lift this up to the top of each test file if you find yourself annoyed at zealous tooling rules:

```js
/*eslint no-unused-expressions: "off"*/


...
```

Reply:
Went one step further and just added a override in the eslint config so that we ignore no-unused-expressions in all test files, does that work?

Reply:
Sure does! :gem: